docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -9,7 +9,7 @@ updatedAt: "2022-07-12T00:00:00.000Z"
 You can download an installer script that will install the Pants binary with this command:
 
 ```
-curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh
```

Comment:
Note that the LHS is arguably buggy since it does not save the file via either `-O`, `-o ...` or via piping. At the least it sends a bit of a confusing message, at least in the broader context of Pants help generally trending towards the over-prescriptive.

Reply:
Anything to do towards this?